### PR TITLE
Changes to members page + blog post

### PR DESCRIPTION
**Pull request by corey-taylor**
Changes include: 

- Jaime, Andrea M, Lisa -> Alumni
- Jaqueline, Sonja, Julian  -> Students
- Student status' updated
- Blog post for digidrug seminar series (digidrug_seminar_series.md)

### CODE REVIEW

**Comment by corey-taylor** [comment]
Any idea why these checks failed? Am I not able to merge from my fork now?

**Comment by corey-taylor** [comment]
Disregard, found some lines it didn't like in the members.yaml. Will edit/push and update the PR.

**Comment by corey-taylor** [comment]
And am I right that the conflict is caused by the fact I had a previous edit to the members.yaml file but that only you can mark it as resolved?

**Comment by dominiquesydow** [comment]
Hi @corey-taylor, I cannot say how this conflict happened - but luckily it seems to be easily resolvable.

> And am I right that the conflict is caused by the fact I had a previous edit to the members.yaml file but that only you can mark it as resolved?

Do I understand correctly that you are not able to resolve this conflict by clicking this button?

image

**Comment by corey-taylor** [comment]
Sorry, my fault it said merging is blocked pending another review. That's what I meant.

**Comment by dominiquesydow** [comment]
Yay, that worked! :) The failing CI is because of the TeachOpenCADD book chapter DOI, which does not work yet. You can ignore it for now - I have it on my list to check up on.

**Comment by dominiquesydow** [comment]
Quick question: When you render the blog post (`hugo serve`), does it show up? For me it doesn't but that might be on me (still investigating).

File: content/blog/digidrug_seminar_series.md
Comment (dominiquesydow) [inline content/blog/digidrug_seminar_series.md:3]: Ah it happens because it is posted in the future, right?

Comment (dominiquesydow) [reply content/blog/digidrug_seminar_series.md:3]: I would set the date to the past, so that you can check the rendering locally.
image

Only very small comment: I would add the key contacts as a list (not in one line).

Comment (corey-taylor) [reply content/blog/digidrug_seminar_series.md:3]: Must be the reason because it doesn't render for me either.

**Comment by AndreaVolkamer** [comment]
@corey-taylor sth is wrong with the team: https://deploy-preview-60--admiring-blackwell-3e6756.netlify.app/team/

you are now Corey and Andrea M. if I go by the figures ;)

**Comment by corey-taylor** [comment]
> @corey-taylor sth is wrong with the team: https://deploy-preview-60--admiring-blackwell-3e6756.netlify.app/team/
> 
> you are now Corey and Andrea M. if I go by the figures ;)

Ah dammit. I'll fix it!

File: content/blog/digidrug_seminar_series.md
Comment (dominiquesydow) [inline content/blog/digidrug_seminar_series.md:11]: Just realized that the last two links are the same.

**Comment by AndreaVolkamer** [comment]
@corey-taylor also with regard to alumni: The students Michele, Allen and Sakshi belong there now too

File: content/blog/digidrug_seminar_series.md
Comment (corey-taylor) [reply content/blog/digidrug_seminar_series.md:11]: Yup, fixed (lazy editing on my part)

File: content/blog/digidrug_seminar_series.md
Comment (corey-taylor) [reply content/blog/digidrug_seminar_series.md:3]: FYI just tried changing the date to yesterday and the blog does render now. So, come tomorrow, it should be visible.

Comment (corey-taylor) [reply content/blog/digidrug_seminar_series.md:3]: With `hugo serve`, I mean.

**Comment by corey-taylor** [comment]
> @corey-taylor also with regard to alumni: The students Michele, Allen and Sakshi belong there now too

Yup, will add them.

**Comment by corey-taylor** [comment]
> > @corey-taylor also with regard to alumni: The students Michele, Allen and Sakshi belong there now too
> 
> Yup, will add them.

Ah that was a change I'd already made today, their status had been wrong for a while.

**Comment by corey-taylor** [comment]
I've pushed a version with Andrea M's status corrected (`member` -> `alumni`) but the Netlify preview still has the wrong version. THis should resolve as soon as you merge, no?

File: data/team/members.yaml
Comment (dominiquesydow) [inline data/team/members.yaml:95]: This bit still renders incorrectly locally (`hugo serve`). Andrea M. has your information right now.

Comment (dominiquesydow) [reply data/team/members.yaml:95]: @AndreaVolkamer do you want her as an alumni already? If so, this bit can be deleted entirely.

Comment (corey-taylor) [reply data/team/members.yaml:95]: > @AndreaVolkamer do you want her as an alumni already? If so, this bit can be deleted entirely.

Yes I just did that. There was duplication.

Comment (dominiquesydow) [reply data/team/members.yaml]: Ok, resolved now!

**Review by dominiquesydow — APPROVED** [review]
Looks good to me!

File: data/team/members.yaml
Comment (AndreaVolkamer) [reply data/team/members.yaml]: @corey-taylor lets keep Andrea M. in for now, until she submitted her thesis

**Comment by AndreaVolkamer** [comment]
@corey-taylor thanks. Looks good to me now.

One small adaption: Mareikes topic needs to be updated: Virtual Screening Pipeline for Drug-Like PPIP5K2 Inhibitors

Then it's ready to be merged from my side!

**Review by AndreaVolkamer — APPROVED** [review]
Looks good to me. Thanks!

**Comment by dominiquesydow** [comment]
CI fails due to some 404 URLs:

[404] public/2021/08/digidrug_seminar_series/Clara.Christ@digidrug.net
[404] public/2021/08/digidrug_seminar_series/Andrea.Volkamer@digidrug.net
[404] public/2021/08/digidrug_seminar_series/Andreas.Bender@digidrug.net
These are ok like this?
If yes, you can maybe skip them like this: https://github.com/volkamerlab/volkamerlab_org/issues/54

[404] https://dx.doi.org/10.1021/bk-2021-1387.ch010
This is our TeachOpenCADD book chapter (https://pubs.acs.org/doi/10.1021/bk-2021-1387.ch010). 
I think the DOI is still not registered but will be valid in the future. I left myself a note here: https://github.com/volkamerlab/volkamerlab_org/issues/61

From my side, this PR can be merged (and I deal with the DOI in another PR).

**Comment by corey-taylor** [comment]
> CI fails due to some 404 URLs:
> 
> [404] public/2021/08/digidrug_seminar_series/Clara.Christ@digidrug.net
> [404] public/2021/08/digidrug_seminar_series/Andrea.Volkamer@digidrug.net
> [404] public/2021/08/digidrug_seminar_series/Andreas.Bender@digidrug.net
> These are ok like this?
> If yes, you can maybe skip them like this: #54
> 
> [404] https://dx.doi.org/10.1021/bk-2021-1387.ch010
> This is our TeachOpenCADD book chapter (https://pubs.acs.org/doi/10.1021/bk-2021-1387.ch010).
> I think the DOI is still not registered but will be valid in the future. I left myself a note here: #61
> 
> From my side, this PR can be merged (and I deal with the DOI in another PR).

I think it just doesn't like those email addresses because they're not links, perhaps. I'll add some tags around them and add the DOI to the Linkinator until the link becomes active.

**Pull request merged**
